Task1.3/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Task1._3
+{
+    internal class Program
+    {
+        static int[] SetCreation(int[] arr)
+        {
+            int count = 0;
```

Comment:
No need to iterate via array twice, first loop is just to calculate the size of final array (set). Better to copy unique values to a temp array with the same size as original, calculate actual length in the same loop as copy operation, and than use the truncated array as resulting